### DISCUSSION TITLE
Is module `ideapad_laptop` supported for the battery threshold feature?

### DISCUSSION THREAD

**Discussion by hueychen27** [AdnanHodzic/auto-cpufreq#748] [Q&A]
I do not have module `ideapad_acpi` for the battery function `auto-cpufreq` has to offer, but I do have `ideapad_laptop`. Will that work as well?

**Comment by PurpleWazard** [comment]
I should but depending on your laptop model is might not work

**Comment by hueychen27** [comment]
I have an Ideapad Slim 5
